ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -22,7 +22,7 @@
       />
     </field.Control>
     {{#if @error}}
```

Comment:
Question: does this `if` statement still display errors if the array happens to be empty?